src/main/java/uk/ac/cam/cl/dtg/isaac/dao/GameboardPersistenceManager.java

```diff
@@ -557,24 +566,34 @@ private GameboardDO saveGameboard(final GameboardDO gameboardToSave)
         try (Connection conn = database.getDatabaseConnection()) {
             pst = conn
                     .prepareStatement("INSERT INTO "
-                            + "gameboards(id, title, questions, wildcard, wildcard_position, "
+                            + "gameboards(id, title, questions, contents, wildcard, wildcard_position, "
                             + "game_filter, owner_user_id, creation_method, tags, creation_date)"
-                            + " VALUES (?, ?, ?, ?::text::jsonb, ?, ?::text::jsonb, ?, ?, ?::text::jsonb, ?);");
-            Array questionIds = conn.createArrayOf("varchar", gameboardToSave.getQuestions().toArray());
-            
+                            + " VALUES (?, ?, ?, ?::text::jsonb[], ?::text::jsonb, ?, ?::text::jsonb, ?, ?,"
+                            + " ?::text::jsonb, ?);");
+
+            List<String> contentsJsonb = Lists.newArrayList();
+            for (GameboardContentDescriptor content: gameboardToSave.getContents()) {
+                contentsJsonb.add(objectMapper.writeValueAsString(content));
+            }
+            Array contents = conn.createArrayOf("jsonb", contentsJsonb.toArray());
+            // TODO MT AUDIENCE_CONTEXT CLEAN UP - Temporarily also store questions in DB for the benefit of old APIs
+            Array questionIds = conn.createArrayOf("varchar",
+                    gameboardToSave.getContents().stream().map(GameboardContentDescriptor::getId).toArray());
```

Comment:
This is to support the old API while it is still about but can then be removed along with the column